sdks/java/core/build.gradle

```diff
@@ -69,7 +67,6 @@ dependencies {
   shadow library.java.vendored_bytebuddy_1_10_8
   shadow library.java.vendored_guava_26_0_jre
   compile library.java.antlr_runtime
-  compile library.java.protobuf_java
```

Comment:
Where does this module get the vendored protobuf dependency?

Reply:
It is a transitive dependency of `:model:pipeline`. Will update to depend on explicitly.